.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+
+jdk: oraclejdk8
+
+before_script:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
This particular command installs the dependencies of the project. So, tests are skipped and Javadoc generation is skipped at that time. So, both commands have a different use.